bindings/core/src/method/mod.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+mod account;
+mod client;
+mod wallet;
+
+use std::fmt::{Formatter, Result as FmtResult};
+
+pub use account::AccountMethod;
+pub use client::ClientMethod;
+use iota_sdk::client::secret::SecretManagerDto;
+pub use wallet::WalletMethod;
+
+pub(crate) trait OmittedDebug {
+    fn omitted_fmt(&self, f: &mut Formatter<'_>) -> FmtResult {
+        f.write_str("<omitted>")
+    }
+}
+impl OmittedDebug for String {}
+impl OmittedDebug for SecretManagerDto {}
+impl<T: OmittedDebug> OmittedDebug for Option<T> {
+    fn omitted_fmt(&self, f: &mut Formatter<'_>) -> FmtResult {
+        match self {
+            Some(_) => f.write_str("Some(<omitted>)"),
```

Comment:
It should print the `OmittedDebug` of the inner T instead of just `<omitted>` because you're potentially not logging information that are not omitted

Reply:
Something like that yes

Reply:
This would be useless, then we only need to use OmittedDebug on T

Reply:
Yeah but you'd lose the Some information